iina-plugin/main.swift

```diff
@@ -0,0 +1,430 @@
+//
+//  plugin.swift
+//  iina-cli
+//
+//  Created by Hechen Li on 5/12/23.
+//  Copyright © 2023 lhc. All rights reserved.
+//
+
+import Foundation
+import Mustache
+
+func printPluginHelp() {
+  print(
+    """
+    Usage:
+        iina-plugin <command> <args>
```

Comment:
It is desirable to follow Posix conventions which can be found [here](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html#tag_12_01). But definitely this should match the conventions of `iina-cli`:
```bash
low-batt@gag MacOS$ ./iina-cli --help
Usage: iina-cli [arguments] [files] [-- mpv_option [...]]
```
Therefore:
- No newline after "Usage:"
- "[arguments]" instead of "\<args\>"

So maybe?:
```text
Usage: iina-plugin <command> [arguments]
```